docs/design/noc-root-cert-design.md

```diff
@@ -15,48 +18,64 @@ To distinguesh NOC root certificates from others, an `isNOC` boolean field will
 ## Transactions
 
 ### 1. ADD_NOC_X509_ROOT_CERTIFICATE
-This transaction adds a NOC Root Certificate owned by the Vendor.
+This transaction adds a NOC root certificate owned by the Vendor.
 
```

Comment:
Please mention, that if a certificate was removed, it should be possible to re-add it.

Reply:
Mentioned in the REMOVE_NOC_X509_ROOT_CERTIFICATE transaction section